src/generate-services.ts

```diff
@@ -288,5 +288,9 @@ export function generateDataLoadersType(): InterfaceSpec {
     .addParameter('identifier', TypeNames.STRING)
     .addParameter('constructorFn', TypeNames.lambda2([], TypeNames.typeVariable('T')))
     .returns(TypeNames.typeVariable('T'));
-  return InterfaceSpec.create('DataLoaders').addModifiers(Modifier.EXPORT).addFunction(fn);
+  return InterfaceSpec.create('DataLoaders').addModifiers(Modifier.EXPORT).addFunction(fn).addProperty('rpcDataLoaderOptions', 'DataLoaderOptions', { optional: true });
```

Comment:
`DataLoaderOptions` is actually https://github.com/graphql/dataloader/blob/6592183a83025c6cbe9b040ea57faf7a2b81bb50/src/index.d.ts#L78 but I'm not sure how to use this type here

Reply:
You can refer to other types like:

```
const dataloader = TypeNames.anyType('DataLoader*dataloader');
```

At the top of this file.

Reply:
Looking at this again, the options type is defined as `Options<K, V>` where `K` & `V` are the request input and output. But since we're setting the options at the context-level I don't think we can really provide these types.

If it's OK, I think I'll just keep this as-is with `DataLoaderOptions` providing a subset of `Options` that can be set at the context-level.